charts/tempo-distributed/values.yaml

```diff
@@ -1747,6 +1807,9 @@ minio:
   persistence:
     size: 5Gi
   resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
```

Comment:
I expect this to be too small to be useful, but I don't have an example to check at the moment.

Reply:
This seems more reasonable

Reply:
How about commenting out the resource section for Minio as well?!

Reply:
Yep, I agree.  I think folks can place their own limits here.  Scale will matter, so choosing sane defaults is a bit difficult.